makefile

```diff
@@ -40,6 +40,9 @@ eunit:
 
 test: compile eunit ct prop
 
+console:
+	erl -pa ebin/ -pa deps/*/ebin
```

Comment:
To `ebin` directory. I just add this target for testing purpose, I use it multiple times while testing amoc.